docs/guides/exporting.md

```diff
@@ -132,6 +132,40 @@ You can also use other tools that work with Jupyter notebooks:
 - [Quarto](https://quarto.org) - Create beautiful documents, websites, presentations
 - [nbgrader](https://nbgrader.readthedocs.io/) - Grade notebook assignments
 
+## Export to WASM-powered HTML
+
+Export your notebook to a self-contained HTML file that runs using WebAssembly:
+
+```bash
+marimo export html-wasm notebook.py -o output_dir --mode run
+# Or edit mode
+marimo export html-wasm notebook.py -o output_dir --mode edit
+```
+
+The exported HTML file will run your notebook using WebAssembly, making it completely self-contained and executable in the browser. This means users can interact with your notebook without needing Python or marimo installed.
+
+Options:
+
+- `--mode`: Choose between `run` (read-only) or `edit` (allows editing)
+- `--output`: Directory to save the HTML and required assets
+
+```{admonition} Note
+:class: note
+
+The exported file must be served over HTTP to function correctly - it cannot be opened directly from the filesystem (file://).
+Your server must also serve the assets in the `assets` directory, next to the HTML file. For this reason, we recommend using the online playground if possible: <https://marimo.app>.
+```
+
```

Comment:
Mention that can be hosted on Github pages?

Reply:
yea i can